.github/drafts/16506/16506_range_slider_draft.md

```diff
@@ -0,0 +1,68 @@
+*** Range Slider ***
+
+The Range Slider widget allows users to choose a range of values from numerical data.
+
+# Properties
+
+Properties allow you to edit the Range Slider widget, connect it with other widgets and customize the user actions.
+
+| **Property** | **Type** | **Description** | **Code Snippet** |
+|--------------|----------|-----------------|------------------|
+| **Min Value** | Formatting | The starting/smallest possible value of the slider. The value may be negative and/or include decimals, however it must always be less than the **Max Value**. _(number)_ | `{{<widget-name>.minValue}}` |
+| **Max Value** | Formatting | The ending/largest possible value of the slider. The value may be negative and/or include decimals, however it must always be greater than the **Min Value**. _(number)_ | `{{<widget-name>.maxValue}}` |
+| **Step Size** | Formatting | The increment by which the user can adjust the slider's value. This increment must be at least 0.1, and can't be less than the **Min Value** or greater than the **Max Value**. _(number)_ | `{{<widget-name>.stepSize}}` |
+| **Min Range** | Formatting | The minimum difference between the Range Slider's starting and ending values. _(number)_ | `{{<widget-name>.minRange}}` |
+| **Default Start Value** | Formatting | This sets the initial value for the slider's lesser/left handle, before the user makes any adjustments. This value should be between the slider's **Min Value** and **Max Value**. _(number)_ | `{{<widget-name>.defaultStartValue}}` |
```

Comment:
The `Default Start Value` must also be greater than the `Default End Value`.

Reply:
@yaldram why will `Default Start Value` be GREATER than `Default end value`?